ruma-events/src/key/verification.rs

```diff
@@ -42,6 +42,8 @@ pub enum KeyAgreementProtocol {
 pub enum MessageAuthenticationCode {
     /// The HKDF-HMAC-SHA256 MAC.
     HkdfHmacSha256,
+    /// The HMAC-SHA256 MAC.
+    HmacSha256,
```

Comment:
The spec doesn't actually mention this, does it?

Reply:
Sure, it's a deprecated method (or perhaps it never made it into the spec, can' remember) but the events are completely useless otherwise, of course unless we resolve https://github.com/ruma/ruma/issues/171.